src/platform_impl/windows/event_loop.rs

```diff
@@ -914,6 +914,41 @@ fn normalize_pointer_pressure(pressure: u32) -> Option<Force> {
     }
 }
 
+enum PointerMoveKind {
+    /// Pointer enterd to the window.
+    Enter,
+    /// Pointer leaved the window client area.
+    Leave,
+    /// Pointer is inside the window or `GetClientRect` failed.
+    None,
+}
+
+unsafe fn get_pointer_move_kind(
```

Comment:
But the function itself is safe. Also move it to the bottom of the file with the `PointerMoveKind` thing.

Reply:
`unsafe` functions in rust serve a different purpose usually, for example, when you need to hold some lifetime, etc invariant when you call a function, which usually is tracked by rust.

In this case though, we know that it as a whole is safe, since the only call uses `HWND` is error checked.